securedrop/tests/test_i18n.py

```diff
@@ -228,7 +228,7 @@ def verify_i18n(app):
             assert 'hreflang="fr-FR"' in locales
 
         c.get("/?l=ar")
-        base = render_template("base.html")
+        base = render_template("error.html", error={})
```

Comment:
This change is worth documenting, I think!

```suggestion
        # We have to render a template that inherits from "base.html" so that "tab_title" will be
        # set.  But we're just checking that when a page is rendered in an RTL language the
        # directionality is correct, so it doesn't matter which template we render.
        base = render_template("error.html", error={})
```